hypertrace-trace-enricher/hypertrace-trace-enricher-api/src/main/java/org/hypertrace/traceenricher/trace/util/GraphBuilderUtil.java

```diff
@@ -23,9 +23,7 @@ static boolean isDifferentTrace(StructuredTrace cachedTrace, StructuredTrace tra
   }
 
   static boolean isStructuredTraceChanged(StructuredTrace cachedTrace, StructuredTrace trace) {
-    return isDifferentTrace(cachedTrace, trace)
-        || isTraceEventsChanged(cachedTrace, trace)
-        || isTraceEntitiesChanged(cachedTrace, trace);
+    return isDifferentTrace(cachedTrace, trace) || isTraceEventsChanged(cachedTrace, trace);
```

Comment:
Do we build `TraceGraph` in multiple places or only ApiTraceGraph?

Reply:
It's being used in other places but does entities should have any change in trace graph? it should be same even if new entities are being added? correct

Reply:
If there is no usage of `TraceEntitiesGraph` from `StructuredTraceGraph`, can we remove that private member from `StructuredTraceGraph`?